apps/gitness/src/pages/webhooks/create-webhook-container.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react'
+import { RepoWebhooksCreatePage, CreateWebhookFormFields } from '@harnessio/playground'
+import { useNavigate } from 'react-router-dom'
+import { useCreateWebhookMutation, CreateWebhookErrorResponse } from '@harnessio/code-service-client'
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+export const CreateWebhookContainer = () => {
+  const repo_ref = useGetRepoRef()
+  const navigate = useNavigate()
+  const [apiError, setApiError] = useState<string | null>(null)
+
+  const { mutate: createWebHook, isLoading: creatingWebHook } = useCreateWebhookMutation(
+    { repo_ref: repo_ref },
+    {
+      onSuccess: () => {
+        setApiError(null)
```

Comment:
don't we get access to `error` directly in `const { mutate, isLoading, error } = useCreate()`?
why do we need to maintain state locally for error?

Reply:
fixed.